modules/nf-scil/registration/synthregistration/main.nf

```diff
@@ -0,0 +1,64 @@
+process REGISTRATION_SYNTHREGISTRATION {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "freesurfer/synthmorph:latest"
+    containerOptions "--entrypoint ''"
+
+    input:
+    tuple val(meta), path(moving), path(fixed)
+
+    output:
+    tuple val(meta), path("*__output_warped.nii.gz"), emit: warped_image
+    tuple val(meta), path("*__deform_warp.nii.gz"), emit: deform_transform
+    tuple val (meta), path("*__init_warp.lta"), emit: init_transform
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    def init = task.ext.init ? "-m " + task.ext.init : "-m affine"
+    def warp = task.ext.warp ? "-m " + task.ext.warp : "-m deform"
+    def header = task.ext.header ? "-H" : ""
+    def threads = task.ext.threads ? "-j " + task.ext.threads : ""
```

Comment:
This should be handled directly in the script with $task.cpus

Reply:
Something like this, for both calls? :

mri_synthmorph -j $task.cpus [...]

Reply:
Yes, exactly